src/fu-plugin.c

```diff
@@ -1555,8 +1555,14 @@ fu_plugin_runner_udev_device_added (FuPlugin *self, FuUdevDevice *device, GError
 	if (func == NULL) {
 		if (priv->device_gtype != G_TYPE_INVALID ||
 		    fu_device_get_specialized_gtype (FU_DEVICE (device)) != G_TYPE_INVALID) {
-			g_debug ("using generic udev_device_added() on %s", priv->name);
-			return fu_plugin_udev_device_added (self, device, error);
+			if (!fu_plugin_udev_device_added (self, device, &error_local)) {
+				if (g_error_matches (error_local,
+						     FWUPD_ERROR,
+						     FWUPD_ERROR_NOT_SUPPORTED))
+					return TRUE;
```

Comment:
maybe we want the same in fu_plugin_runner_usb_device_added?

Reply:
Yes, just added.